### ISSUE TITLE
Edge browse mode: When moving to next or previous heading, NVDA does not announce that you have moved to a heading and its level unless heading levels change

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5254]
**Reported by nvdakor on 2015-07-31 20:40**
Hi,
This may turn out to be start of a train of bug reports for Microsoft Edge:
When moving by heading (H and Shift+H), NVDA does not announce that one has moved to a heading or heading level unless a user moves to a heading with a different heading level. This isn't the case with Internet Explorer (including one that comes with Windows 10) and Firefox 39.
STR:
1. If using Windows 10, open Internet Explorer 11 and Microsoft Edge to a site with lots of headings, such as Facebook timeline, a Wikipedia article and so on. If using Firefox, open the same page as well.
2. Press H to move forward by heading.
Expected: NVDA announces heading element and its level.
Actual: In Microsoft Edge, heading element and its level is not announced unless one moves to heading with different heading levels.
Note that the actual heading text is announced in all browsers (Edge included).
Setup:
- OS: Windows 10 Build 10240
- NVDA version: next.12368 (presumably this happens in master as well)
- Browsers: IE 11, Microsoft Edge (RTM version), Firefox 39.
  Thanks.

**Comment by michaelDCurran** [comment]
Fixed in PR #6271
